### PR TITLE
Add Discord to contact informations

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR
- creates a dedicated `Communication channels` page
- adds the new Discord server invitation link to the page
- updates all other places

/cc @schalkneethling

### CODE REVIEW

**Comment by OnkarRuikar** [comment]
Changes have been made considering Discord as the main choice for direct communication with the team/community.

If we want Discourse can be replaced by [Discord forum channels](https://support.discord.com/hc/en-us/articles/6208479917079-Forum-Channels-FAQ):
code_help_forums

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ A pull request represents the work you want to be reviewed, approved, and merged
 See the [Creating a pull request](#creating-a-pull-request) for more details on creating and handling pull requests successfully.
 
 If you're not certain of the changes that you want to make, get in touch with us!
-You can [chat with us](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [file an issue](https://github.com/mdn/content/issues).
+You can chat with us on [community Discord server](https://discord.gg/apa6Rn7uEj) or [file an issue](https://github.com/mdn/content/issues).
```
Comment (hamishwillee) [inline CONTRIBUTING.md:92]:
```suggestion
You can chat with us on the [community Discord server](https://discord.gg/apa6Rn7uEj) or [file an issue](https://github.com/mdn/content/issues).
```

File: README.md [hunk 2]
```diff
@@ -44,4 +44,4 @@ By participating in and contributing to our projects and discussions, you acknow
 
 ## Get in touch
 
-You can communicate with the MDN Web Docs team and community through discussions on [GitHub](https://github.com/mdn/mdn-community/discussions), our forums on [Discourse](https://discourse.mozilla.org/c/mdn/236), and a [Matrix channel](https://wiki.mozilla.org/Matrix).
+You can communicate with the MDN Web Docs team and community through discussions on [community Discord server](https://discord.gg/apa6Rn7uEj), [GitHub discussions](https://github.com/mdn/mdn-community/discussions), or using our forums on [Discourse](https://discourse.mozilla.org/c/mdn/236).
```
Comment (hamishwillee) [inline README.md:47]:
```suggestion
You can communicate with the MDN Web Docs team and community through discussions on the [community Discord server](https://discord.gg/apa6Rn7uEj), [GitHub discussions](https://github.com/mdn/mdn-community/discussions), or using our forums on [Discourse](https://discourse.mozilla.org/c/mdn/236).
```

File: REVIEWING.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ inappropriate, than it is to follow every guideline to the letter.
    assigned already. In such cases, the PR should be linked to an issue
    that explains all these details. If you are not sure, ask the submitter
    if they need a review of the content, and where the rationale behind the
-   change is explained. Ping our team on [MDN Web Docs chat room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) to ask for help if you are still not sure, or
+   change is explained. Ping our team on [MDN Web Docs community Discord server](https://discord.gg/apa6Rn7uEj) to ask for help if you are still not sure, or
```
Comment (hamishwillee) [inline REVIEWING.md:103]:
```suggestion
   change is explained. Ping our team on the [MDN Web Docs community Discord server](https://discord.gg/apa6Rn7uEj) to ask for help if you are still not sure, or
```

File: files/en-us/learn/javascript/first_steps/what_went_wrong/index.md [hunk 4]
```diff
@@ -267,7 +267,7 @@ So there we have it, the basics of figuring out errors in simple JavaScript prog
 ## See also
 
 - There are many other types of errors that aren't listed here; we are compiling a reference that explains what they mean in detail — see the [JavaScript error reference](/en-US/docs/Web/JavaScript/Reference/Errors).
-- If you come across any errors in your code that you aren't sure how to fix after reading this article, you can get help! Ask for help on the [MDN Discourse forum Learning category](https://discourse.mozilla.org/c/mdn/learn/250), or in the [MDN Web Docs room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on [Matrix](https://wiki.mozilla.org/Matrix). Tell us what your error is, and we'll try to help you. A listing of your code would be useful as well.
+- If you come across any errors in your code that you aren't sure how to fix after reading this article, you can get help! Ask for help on [community Discord server](https://discord.gg/apa6Rn7uEj). Tell us what your error is, and we'll try to help you. A listing of your code would be useful as well.
```
Comment (hamishwillee) [inline files/en-us/learn/javascript/first_steps/what_went_wrong/index.md:270]:
```suggestion
- If you come across any errors in your code that you aren't sure how to fix after reading this article, you can get help! Ask for help on the [community Discord server](https://discord.gg/apa6Rn7uEj). Tell us what your error is, and we'll try to help you. A listing of your code would be useful as well.
```

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 5]
```diff
@@ -12,13 +12,15 @@ tags:
 
 We are an open community of developers, technical writers, and learners building resources for a better Web, regardless of brand, browser, or platform. Anyone can contribute, and each person who does contribute makes us stronger. Together we can continue to drive innovation on the Web to serve the greater good. It starts here, with you.
 
-[Join us!](https://github.com/mdn/mdn-community/)
+Join us on
+[Community Discord server](https://discord.gg/apa6Rn7uEj)
+[GitHub discussions](https://github.com/mdn/mdn-community/)
```
Comment (hamishwillee) [inline files/en-us/mdn/community/contributing/getting_started/index.md:17]:
```suggestion
Join us on the [Community Discord server](https://discord.gg/apa6Rn7uEj) and [GitHub discussions](https://github.com/mdn/mdn-community/)
```

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 6]
```diff
@@ -12,13 +12,15 @@ tags:
...
 
 ## What can I do to help?
 
 There are multiple avenues you can take to contribute to MDN, depending on your skill set and interests. Therefore, along with each task, we provide a short description and an approximate time each type of task typically takes.
 
-> If unsure what to do, you are always welcome to [ask for help](https://github.com/mdn/mdn-community/).
+> If unsure what to do, you are always welcome to [ask for help on community Discord server](https://discord.gg/apa6Rn7uEj).
```
Comment (hamishwillee) [inline files/en-us/mdn/community/contributing/getting_started/index.md:23]:
```suggestion
> If unsure what to do, you are always welcome to [ask for help on the community Discord server](https://discord.gg/apa6Rn7uEj).
```

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 7]
```diff
@@ -52,4 +54,4 @@ This ensures that two people do not work on the same issue, and you will know wh
 
 When contributing, you agree to make your contributions available under the [Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/4.0/) (or an alternative license already specified by the page you are editing). In addition, code samples are available under [Creative Commons CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/) (a Public Domain dedication).
 
-> If you have any questions or concerns about anything discussed here, please [open a discussion](https://github.com/mdn/mdn-community/discussions/categories/content) and let us know.
+> If you have any questions or concerns about anything discussed here, please [contact us on community Discord server](https://discord.gg/apa6Rn7uEj) or [open a discussion](https://github.com/mdn/mdn-community/discussions/categories/content).
```
Comment (hamishwillee) [inline files/en-us/mdn/community/contributing/getting_started/index.md:57]:
```suggestion
> If you have any questions or concerns about anything discussed here, please [contact us on the community Discord server](https://discord.gg/apa6Rn7uEj) or [open a discussion](https://github.com/mdn/mdn-community/discussions/categories/content).
```

File: files/en-us/mdn/community/contributing/translated_content/index.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ We have frozen all localized content (meaning that we won't accept any edits to
 
 ## Active locales
 
-> **Note:** If you want to contribute to one of the existing active locales get in touch with one of the active members listed below, or [contact us for help](/en-US/docs/MDN/Community/Contributing/Getting_started#step_4_ask_for_help).
+> **Note:** If you want to contribute to one of the existing active locales get in touch with one of the active members listed below, or contact us for help on [community Discord server](https://discord.gg/apa6Rn7uEj).
```
Comment (hamishwillee) [inline files/en-us/mdn/community/contributing/translated_content/index.md:19]:
```suggestion
> **Note:** If you want to contribute to one of the existing active locales get in touch with one of the active members listed below, or contact us for help on the [community Discord server](https://discord.gg/apa6Rn7uEj).
```

**Review by hamishwillee — DISMISSED** [review]
I've added a bunch of comments - unless it was intentional isn't it "THE community Discord server"? If so, fix everywhere.

Looking at all these links we have the following comms channels
- discourse
- chat rooms
- Discord
- MDN discussions
- Etc.

That seems like a stupidly long list. In any case, I'd create a "Communication channels" or "Support" section somewhere and change most of these links to "If you want to discuss this further see". Up to you, but that will ease long term maintenance.

I'd put Discord at the bottom because it is the ~least usable~ one that is going to be hardest to search as a FAQ for long term storage of "answers", or possibly describe alongside each what sorts of discussions they are intended for.

**Comment by OnkarRuikar** [comment]
> In any case, I'd create a "Communication channels" or "Support" section somewhere and change most of these links to

That is a good idea! Clearly these are way too many pages to change contact info every time. When we hit 5k users mark on the server we'll get custom invite link(without numbers) like `https://discord.gg/mdn-web-docs`.
I'll create a separate page for all contact info.

> I'd put Discord at the bottom because it is the least usable one that is going to be hardest to search as a FAQ for long term storage of "answers", or possibly describe alongside each what sorts of discussions they are intended for.

But discord is going to have fastest response time and more activity becasue all other channel(matrix, twitter, mastodon etc.) members have joined it. Waiting on @schalkneethling for deciding priority of channels.

**Comment by hamishwillee** [comment]
> But discord is going to have fastest response time and more activity because all other channel(matrix, twitter, mastodon etc.) members have joined it. Waiting on @schalkneethling for deciding priority of channels.

This may make sense. In most communities in which I'm a member permanent discussions are valued for their reusability/searchability - in particular over slack which you have to pay for to store the data. It may be that we don't care about historical data though, so we'll funnel everyone to discord.

There is a lot to be said for "just one channel" for comms.

**Comment by OnkarRuikar** [comment]
> It may be that we don't care about historical data though, so we'll funnel everyone to discord.

For historical data, I think we have GitHub discussions we mostly take big decisions there.

With current setup discussions are happening in different places, and certainly we are not keeping track of that data. We do need to assert that which channel is for what purpose.

File: CONTRIBUTING.md (hunk 1 above)
Comment (OnkarRuikar) [reply CONTRIBUTING.md:92]: going to create a new page for all the links

**Comment by bsmth** [comment]
>I'd create a "Communication channels" or "Support" section somewhere and change most of these links to "If you want to discuss this further see". Up to you, but that will ease long term maintenance.

I agree with this, we should take stock of places we list the different communication channels and I would `+1` on a single page that summarizes how to get in touch via these channels and their purpose.

**Comment by OnkarRuikar** [comment]
I've made the changes. Added a dedicated page named [Communication channels](https://pr23789.content.dev.mdn.mozit.cloud/en-US/docs/MDN/Community/Communication_channels), and updated content documents to refer to it. Let me know how it looks.

File: files/en-us/mdn/community/communication_channels/index.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+- governance
+---
+
+{{MDNSidebar}}
+
+There are various communication channels using which community can contact MDN Web Docs staff and participate in discussions.
+
+## Chat rooms
+
+Chat rooms are for queries, clarifications, sharing information, and meetings.
```
Comment (bsmth) [inline files/en-us/mdn/community/communication_channels/index.md:17]: Adding one or two sentences for context 👍🏻

```suggestion
Chat rooms are for queries, clarifications, sharing information, and meetings.
They are also a great place to get to know other community members, share your work, and get help with your contributions.
If you have a question or want to get in touch with us about something, feel free to join us.
```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+### Discord server
+
```
Comment (bsmth) [inline files/en-us/mdn/community/communication_channels/index.md:20]: Minor hint on the timing as it shows it's fresh

```suggestion
As of January 2023, MDN Web Docs opened up a Discord server to the public.
This server is a great place to see what staff and members of the community are doing on a daily basis and to find out how you can get involved.

```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+- [MDN Web Docs community discord server](https://discord.gg/apa6Rn7uEj)
+
+### Matrix chat rooms
+
```
Comment (bsmth) [inline files/en-us/mdn/community/communication_channels/index.md:24]:
```suggestion
The Matrix chat rooms are an alternative to the Discord server.

```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+- [MDN room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on Mozilla Element web app
+- [MDN room](https://app.element.io/#/room/#mdn:mozilla.org) on Matrix Element web app
+- [Add-ons room](https://chat.mozilla.org/#/room/#addons:mozilla.org)
+- [List of all Mozilla rooms](https://wiki.mozilla.org/Matrix#Commonly_used_rooms)
+
+## GitHub Discussions
+
+[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work. Refer [the guidelines](/en-US/docs/MDN/Community/Discussions) before opening a discussion.
```
Comment (bsmth) [inline files/en-us/mdn/community/communication_channels/index.md:32]:
```suggestion
[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work.
You can use GitHub discussions if you think the answer to your question might be helpful for other people to find in the future.
See [the discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.
```

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 13]
```diff
@@ -12,14 +12,14 @@ tags:
...
-[Join us!](https://github.com/mdn/mdn-community/)
+[Join us!](/en-US/docs/MDN/Community/Communication_channels)
 
 ## What can I do to help?
 
 There are multiple avenues you can take to contribute to MDN, depending on your skill set and interests. Therefore, along with each task, we provide a short description and an approximate time each type of task typically takes.
 
-> If unsure what to do, you are always welcome to [ask for help](https://github.com/mdn/mdn-community/).
-> Also note that our small, but mighty docs team maintains this repo, to preserve our bandwidth, off topic conversations will be closed.
+> If unsure what to do, you are always welcome to ask for help on the [communication channels](/en-US/docs/MDN/Community/Communication_channels).
```
Comment (bsmth) [inline files/en-us/mdn/community/contributing/getting_started/index.md:21]:
```suggestion
> If you're unsure what to do, you can always ask for help in one of [our communication channels](/en-US/docs/MDN/Community/Communication_channels).
```

File: files/en-us/mdn/community/roles_teams/index.md [hunk 14]
```diff
@@ -320,8 +319,8 @@ The teams in the [MDN GitHub organization](https://github.com/orgs/mdn/teams) in
 - `@sre`: Site reliability engineers who support MDN Web Docs
 - `@yari-content`: The umbrella team for all MDN Web Docs content reviewers
 
-  - There is a sub-team for the different topic areas — accessibility, Add-ons, CSS, HTML, HTTP, JavaScript, SVG, Web API, and WebAssembly. For example, there's `@yari-content-css` and `@yari-content-svg`.
-  - There are also sub-teams for different languages — Brazilian Portuguese, Chinese, French, Japanese, Korean, Russian, and Spanish. For example, there's `@yari-content-fr` and `@yari-content-ko`.
+  - There is a subteam for the different topic areas — accessibility, Add-ons, CSS, HTML, HTTP, JavaScript, SVG, Web API, and WebAssembly. For example, there's `@yari-content-css` and `@yari-content-svg`.
```
Comment (bsmth) [inline files/en-us/mdn/community/roles_teams/index.md:322]: TIL

File: files/en-us/mdn/writing_guidelines/howto/document_an_http_header/index.md [hunk 15]
```diff
@@ -18,7 +18,7 @@ You will need to know or be able to dive into some [HTTP](/en-US/docs/Web/HTTP).
 - There are many HTTP headers defined in various IETF standards.
 - IANA maintains a [registry of headers](https://www.iana.org/assignments/message-headers/message-headers.xhtml) and Wikipedia lists the [known header fields](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields), but not all are relevant to web developers or are part of an official standard.
 - If there are any **red links** on the current [HTTP headers reference overview page](/en-US/docs/Web/HTTP/Headers), these headers are a good choice to document.
-- If in doubt, ask the [MDN Web Docs team](https://github.com/mdn/mdn-community/discussions) whether or not it makes sense to write about the header you have chosen.
+- If in doubt, ask the MDN Web Docs team on the [MDN Web Docs chat rooms](/en-US/docs/MDN/Community/Communication_channels#chat_rooms) whether or not it makes sense to write about the header you have chosen.
```
Comment (bsmth) [inline files/en-us/mdn/writing_guidelines/howto/document_an_http_header/index.md:21]:
```suggestion
- If in doubt, [ask the MDN Web Docs team](/en-US/docs/MDN/Community/Communication_channels) whether or not it makes sense to write about the header you have chosen.
```

**Review by bsmth — APPROVED** [review]
Thanks, @OnkarRuikar, I like the changes, well done. Going to ask for another `+1` in our daily later in case there's anything we need to change on the Communication Channels page.

File: files/en-us/mdn/community/communication_channels/index.md (hunk 9 above)
Comment (OnkarRuikar) [reply files/en-us/mdn/community/communication_channels/index.md:17]: > Adding one or two sentences for context 👍🏻

```suggestion
Chat rooms are for queries, clarifications, sharing information, and meetings.
They are also great places to get to know other community members, share your work, and get help for your contributions.
If you have a question or want to get in touch with us about something, feel free to join us.
```

File: files/en-us/mdn/community/pull_requests/index.md [hunk 16]
```diff
@@ -35,7 +35,7 @@ The content changes go through the following stages before they are published on
 We want MDN Web Docs to be a welcoming, friendly community that we can all be proud of.
 All participants must follow our [Code of Conduct](https://github.com/mdn/content/blob/main/CODE_OF_CONDUCT.md) which means adhering to [Mozilla's Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
 Be polite and constructive when opening pull requests, writing review comments, interacting with the pull request author or other community members.
-If anyone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).
+If anyone has engaged in behavior that is potentially illegal or makes you or someone else feels unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/mdn/community/pull_requests/index.md:38]: > I think the original is correct here :)

My bad didn't read the full sentence :( Rephrasing it to make it direct.
```suggestion
If you or someone else has experienced behavior that is potentially illegal or makes you feel unsafe, unwelcome, or uncomfortable, we encourage you to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).
```

[the remark this answers is not in the document]
Comment (akasyagroup) [reply files/en-us/mdn/community/pull_requests/index.md]: @akasyaagroup.com

**Comment by dipikabh** [comment]
Thanks @OnkarRuikar for these updates! I'll take a quick look.

File: files/en-us/mdn/community/communication_channels/index.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+- [MDN room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on Mozilla Element web app
+- [MDN room](https://app.element.io/#/room/#mdn:mozilla.org) on Matrix Element web app
+- [Add-ons room](https://chat.mozilla.org/#/room/#addons:mozilla.org)
+- [List of all Mozilla rooms](https://wiki.mozilla.org/Matrix#Commonly_used_rooms)
+
+## GitHub Discussions
+
+[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work.
+You can use GitHub Discussions if you think the answer to your question might be helpful for other people to find in the future.
+See [the Discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.
```
Comment (dipikabh) [inline files/en-us/mdn/community/communication_channels/index.md:41]: I began with a few edits and ended up rewriting a whole lot. Let me know what you think.

```suggestion
[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) on MDN Web Docs is the place to look out for any project-wide [announcements](https://github.com/mdn/mdn-community/discussions/categories/announcements). 

Use GitHub discussions to post your questions and propose your suggestions. Choose the appropriate [discussion category](https://github.com/mdn/mdn-community#github-discussions) when starting a thread. Your answered questions can be really useful for other people looking for similar information in the future.

We strongly recommend you to use GitHub Discussions to share the work you're planning to do and that you think will need a consensus on the course of action. You can also use GitHub Discussions for brainstorming your ideas to define an actionable piece of work, especially when the changes have a wide-ranging impact.

Check out the MDN-specific [discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.
```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+## Forums
+
+The forums can be used for discussing code problems, assessment with the skill tests, assessments that appear on the Learn web development section on MDN, and if you want someone to take a look at your page.
```
Comment (dipikabh) [inline files/en-us/mdn/community/communication_channels/index.md:45]:
```suggestion
You can use the forums listed below for discussing code problems. You can also request feedback here on your progress with the assessments in the [MDN learning area](https://developer.mozilla.org/en-US/docs/Learn).
```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+- [MDN Discourse forum](https://discourse.mozilla.org/c/mdn/236)
+- [MDN Discourse forum learning category](https://discourse.mozilla.org/c/mdn/learn/250)
+- [Add-ons Discourse](https://discourse.mozilla.org/c/add-ons/35)
+
+## Localization channels
+
+Each localization team has its own [methods of communication](/en-US/docs/MDN/Community/Contributing/Translated_content).
```
Comment (dipikabh) [inline files/en-us/mdn/community/communication_channels/index.md:53]:
```suggestion
Each localization team has its own [method of communication](/en-US/docs/MDN/Community/Contributing/Translated_content).
```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+
+## Bugs and issues
```
Comment (dipikabh) [inline files/en-us/mdn/community/communication_channels/index.md:55]: This is not a communication channel per se and should not be listed here. I think for this guidance, folks will need to refer to https://developer.mozilla.org/en-US/docs/MDN/Community/Issues. But this is a good list, maybe we can move it to https://developer.mozilla.org/en-US/docs/MDN/Community/Issues#reporting_an_issue?

File: files/en-us/mdn/community/communication_channels/index.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+- [mdn/translated-content](https://github.com/mdn/translated-content/issues)
+- [yari](https://github.com/mdn/yari/issues)
+- [browser-compat-data](https://github.com/mdn/browser-compat-data/issues)
+- [interactive-examples](https://github.com/mdn/interactive-examples/issues)
+- [dom-examples](https://github.com/mdn/dom-examples/issues)
+- [learning-area](https://github.com/mdn/learning-area/issues)
+
+## Mailing list
+
+For nonpublic communication email the [mdn-admins list](mailto:mdn-admins@mozilla.org).
```
Comment (dipikabh) [inline files/en-us/mdn/community/communication_channels/index.md:69]:
```suggestion
For any nonpublic communication, send an email to [mdn-admins](mailto:mdn-admins@mozilla.org).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
I've left a few edits/suggestions on the new Communication channels page :)

File: files/en-us/mdn/community/communication_channels/index.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+page-type: mdn-community-guide
+tags:
+- meta
+- community-guidelines
+- governance
+---
+
+{{MDNSidebar}}
+
+There are various communication channels using which community can contact MDN Web Docs staff and participate in discussions.
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/mdn/community/communication_channels/index.md:13]:
```suggestion
The community can use one of the many communication channels described here to contact the MDN Web Docs staff, ask questions and participate in discussions.
```

File: files/en-us/mdn/community/communication_channels/index.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+## Chat rooms
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/mdn/community/communication_channels/index.md:15]: > Should we order these channels in the order of recommended use? So say, the 'Discord server' first, 'GitHub Discussions' next, and so on? Possibly move Matrix toward the end of the list?

I've grouped them by type(chat, github, forums, email etc.) and sorted based on frequency of use/importance.
references
Looks like our current stance is that discord is in experimentation phase and we are not leaving matrix till everything goes well with discord. So they are on same recommendation level.
Discord has many features so it'll definitely going to be prominent in future.

File: files/en-us/mdn/community/communication_channels/index.md (hunk 20 above)
Comment (OnkarRuikar) [reply files/en-us/mdn/community/communication_channels/index.md]: done

**Comment by OnkarRuikar** [comment]
Do you think we need to update the website footer? if yes, how?
references

The footer is using Matrix chat at the moment.

**Comment by dipikabh** [comment]
> Do you think we need to update the website footer?

Probably too early to update. We're not sunsetting Matrix update. Let's see how Discord plays out with the community over the coming months.

>  if yes, how?

The code is most likely in yari

**Review by dipikabh — APPROVED** [review]
The updates look good. Thanks a lot @OnkarRuikar for your contribution to add the new page!

**Comment by hamishwillee** [comment]
> I believe all of Hamish's comments have been addressed.

Yes, this looks excellent. Thanks @OnkarRuikar.

**Pull request merged**